web/client/main.js

```diff
@@ -155,17 +183,33 @@ function start(isInitiator) {
             pc.createOffer(localDescCreated, logError);
     };
 
+    // start the chat
+    if (chatCheckBox.checked) {
+        if (isInitiator) {
+            channel = pc.createDataChannel("chat");
+            setupChat();
+        } else {
+            pc.ondatachannel = function (evt) {
+                channel = evt.channel;
+                setupChat();
+            };
+        }
+    }
+
     // once the remote stream arrives, show it in the remote video element
     pc.onaddstream = function (evt) {
         remoteView.src = URL.createObjectURL(evt.stream);
         if (videoCheckBox.checked)
             remoteView.style.visibility = "visible";
-        else
+        else if (audioCheckBox.checked) {
             audioOnlyView.style.visibility = "visible";
-        sendOrientationUpdate();
+            sendOrientationUpdate();
```

Comment:
`sendOrientationUpdate();` should not be inside brackets. With this, orientation will only be sent when audio is enabled, rather than for both audio and video.

Move `sendOrientationUpdate();` to the end of the `pc.onaddstream = function (evt) {}` method.